contributing/code_of_conduct_concrete_example_document.rst

```diff
@@ -0,0 +1,30 @@
+Code of Conduct: Concrete Example Document
+==========================================
+
+This is a living document that serves to give concrete examples to
```

Comment:
to => of ?

Reply:
Not sure "of" is right in this case

Reply:
I would say "of" is the correct preposition in this case.

Reply:
Yeah, of is better. Changed it. Thanks! :)